schema/schema.yaml

```diff
@@ -589,4 +589,89 @@ tables:
         expression: radial_velocity_error_kms >= 0
 
 
-version: "v0.1"
\ No newline at end of file
+  - name: CompanionRelationships
+    "@id": "#CompanionRelationships"
+    description: Relationships between sources
+    primaryKey: 
+    - "#CompanionRelationships.source"
+    - "#CompanionRelationships.companion" # or companion_name
+
+    columns:
+    - name: source
+      "@id": "#CompanionRelationships.source"
+      datatype: string
+      length: 100
+      description: Main identifier for an object; links to Sources table
+      ivoa:ucd: meta.id;meta.main
+      nullable: false
+    - name: companion
+      "@id": "#CompanionRelationships.companion"
+      datatype: string
+      length: 100
+      description: Identifier for a companion object
+      ivoa:ucd: meta.id
+      nullable: false
+    - name: relationship
+      "@id": "#CompanionRelationships.relationship"
+      datatype: string
+      length: 30
+      description: Relationship of the source to the companion, e.g., "parent", "child", "sibling"
+      nullable: false
+    - name: projected_separation_arcsec
+      "@id": "#CompanionRelationships.projected_separation_arcsec"
+      datatype: double
+      description: Projected separation between the source and companion in arcseconds
+      fits:tunit: arcsec
+      ivoa:ucd: pos.angDistance
+    - name: projected_separation_error_arcsec
+      "@id": "#CompanionRelationships.projected_separation_error_arcsec"
```

Comment:
For the sake of shorter column names, since you already have `projected_separation_arcsec`, could this just be `projected_separation_error`? That is- assume the same units as the value?

Reply:
Agreed